addons/mail/static/tests/discuss/call/web/call.test.js

```diff
@@ -44,12 +43,11 @@ test("no default rtc after joining a group conversation", async () => {
     pyEnv["res.users"].create([{ partner_id: partnerId_1 }, { partner_id: partnerId_2 }]);
     await start();
     await openDiscuss();
-    await click(".o-mail-DiscussSidebar [title='Start a conversation']");
+    await insertText(".o-mail-DiscussSearch input", "mario");
```

Comment:
Prefer to target `input[placeholder=""]`, as this is relevant for end-users. They do not see classname `o-mail-DiscussSearch`, this is more technical stuff that should better not be used in tests.

We overused them, and sometimes for complex components it's fine to target the root classname. When we can avoid targeting structural classnames and instead target exact text node or `title` or `aria-label` or icons, then this is much closer to what the end-user expects so test is far more maintainable.